plugin/pkg/scheduler/factory/factory.go

```diff
@@ -274,9 +276,16 @@ func (factory *ConfigFactory) makeDefaultErrorFunc(backoff *podBackoff, podQueue
 		// Note that this is extremely rudimentary and we need a more real error handling path.
 		go func() {
 			defer util.HandleCrash()
-			podID := pod.Name
+			podID := pod.Namespace + "/" + pod.Name
```

Comment:
To be super pedantic, there's a `cache.MetaNamespaceKeyFunc` which is what all the fifo/cache stuff uses.